.github/zizmor.yml

```diff
@@ -0,0 +1,5 @@
+rules:
+  unpinned-uses:
+    ignore:
+      - style_docstr.yml:78:9
```

Comment:
This exception ignores the following audit rule error:

```
error[unpinned-uses]: unpinned action reference
  --> .github/workflows/style_docstr.yml:78:9
   |
78 |         uses: errata-ai/vale-action@reviewdog
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ action is not pinned to a hash (required by blanket policy)
   |
   = note: audit confidence → High
```

The offending `errata-ai/vale-action` GHA appears to be referencing the `reviewdog` branch rather than a released version or tag ... so didn't quite know what to do about this. The alternative would be to pin to a commit SHA on that branch?

Reply:
@MatthewFlamm Done :+1: 

I've pinned this to the latest commit SHA on the `reviewdog` branch.